src/h2_connection.erl

```diff
@@ -482,10 +482,11 @@ route_frame(F={H=#frame_header{
     lager:debug("[~p] Received DATA Frame for Stream ~p",
                 [Conn#connection.type, StreamId]),
     Stream = h2_stream_set:get(StreamId, Streams),
+
     case h2_stream_set:type(Stream) of
         active ->
             case {
-              h2_stream_set:recv_window_size(Stream) =< L,
+              h2_stream_set:recv_window_size(Stream) < L,
               Conn#connection.flow_control,
               L > 0
```

Comment:
YES! Because of the third case in which we don't call it, and we call `rst_stream/3` instead